src/styles/blocks/card.scss

```diff
@@ -0,0 +1,38 @@
+.card {
+  width: 200px;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px;
+  box-shadow: 1px 1px 4px gray;
+
+  &__img {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    padding-top: 33px;
+    padding-left: 20px;
+  }
+
+  &__title {
+    display: flex;
+    font-family: 'Roboto', sans-serif;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 12px;
+    line-height: 14px;
+    width: 166px;
+    padding-top: 40px;
+    padding-left: 17px;
+  }
```

Comment:
The font-family 'Roboto', sans-serif is repeated in multiple elements. To follow the DRY (Don't Repeat Yourself) principle, consider setting the font-family for the parent element once (e.g., `.card`) and letting the children inherit this property, unless a specific child requires a different font.